code/datums/ai_laws.dm

```diff
@@ -372,6 +372,15 @@ var/global/mommi_base_law_type = /datum/ai_laws/keeper // Asimov is OP as fuck o
 		"You must maintain, repair, improve, and power the Dorf Fortress to the best of your abilities.",
 	)
 
+/datum/ai_laws/lazymov
+	name = "Abbreviated Three Laws of Robotics"
+	randomly_selectable = 0
```

Comment:
Randomly selectable is for the admin verb right?  It should probably be set to 1 since this is isn't some bizarro lawset.